mobile/src/components/common/table/table.tsx

```diff
@@ -10,20 +10,22 @@ type Props<Data extends Record<string, unknown>> = {
   columns: TableColumn<Data>[];
   data: Data[];
   columnWidthArr?: number[];
+  columnWidth?: number;
 };
 
 const Table = <Data extends Record<string, unknown>>({
   columns,
   data,
-  columnWidthArr,
+  columnWidth,
+  columnWidthArr = columns.map(() => (columnWidth ? columnWidth : 150)),
 }: Props<Data>): ReactElement => {
   const headers = columns.map(({ header }) => header);
   const tableData = data.map((entry) =>
     columns.map(({ accessor }) => entry[accessor]),
   );
 
   return (
-    <ScrollView horizontal={true}>
+    <ScrollView horizontal={true} contentContainerStyle={styles.scrollView}>
```

Comment:
Why do we need to scroll views in table?

Reply:
> Why do we need to scroll views in table?

![image](https://raw.githubusercontent.com/Gil2015/tools_file/master/img/react-native-table-component/exampleThree.gif)

Reply:
We need only horizontal scroll in table component, vertical scroll we will add for screens

Reply:
agreed